RFC/src/RFC-1000_TariUseCases.md

```diff
@@ -0,0 +1,187 @@
+![status: draft](https://github.com/tari-project/tari/raw/master/RFC/src/theme/images/status-draft.svg)
+
+**Maintainer(s)**: [Leland Lee](https://github.com/lelandlee)
+
+# License
+
+[ The 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause).
+
+Copyright 2019 The Tari Development Community
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+following conditions are met:
+
+1. Redistributions of this document must retain the above copyright notice, this list of conditions and the following
+   disclaimer.
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following
+   disclaimer in the documentation and/or other materials provided with the distribution.
+3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products
+   derived from this software without specific prior written permission.
+
+THIS DOCUMENT IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+There will be many types of digital assets that can be issued on Tari. This document is intended to help potential asset
+issuers identify use cases for Tari that lead to the design of new types of digital assets that may or may not exist in
+their ecosystems today.
+
+## Related RFCs
+* [RFC-0001: An overview of the Tari network](RFC-0001_overview.md)
+* [RFC-0300: The Digital Assets Network](RFC-0300_DAN.md)
+
+## Description
+
+
+### Types
+A potential high level taxonomy of different types of digital assets.
```

Comment:
I don't think this is intended to be a specification, rather just an ideation document (i.e. giving people ideas that they can build on top of Tari)

Reply:
@hansieodendaal I see your point. Will write this document up as a specification but note that is it meant to be an informal framework for now at the top.

Reply:
I actually feel that this doc is NOT a spec. I feel that Writing it as one will make the DAN feel prescriptive, like these are the rules of what is/isn't valid.

I see it more as a source of ideas for the kind of cool things people could build on top of Tari

Reply:
I actually feel that this doc is NOT a spec. I feel that Writing it as one will make the DAN feel prescriptive, like these are the rules of what is/isn't valid.

I see it more as a source of ideas for the kind of cool things people could build on top of Tari

Reply:
If this is not a spec, is there a formal term for it rather than ideation framework? Seems like we should not use BCP14 words, is there particular terms we should use or avoid?

Reply:
@CjS77 and @lelandlee, this can be simple then. If not a specification, all capitalized MAY and SHOULD must just become lower case. The document states:

`The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 (covering RFC2119 and RFC8174) when, and only when, they appear in all capitals, as shown here.`

I updated all my comments to make use of the conversational tone and not the capitalized terms.

Reply:
Thanks for clarifying. I've gone through and merged the changes.